src/process/terminal_source.rs

```diff
@@ -23,7 +21,7 @@ pub struct ColorableTerminal {
     // Arc<Mutex<...> which isn't great, but OTOH it is test code. Locking the
```

Comment:
These comments explains that the original code was written like so due to a limitation in `TermColor` which seems to be long gone. Maybe this can be removed as well?

Reply:
Yeah, I don't think this makes sense to keep. Removed.